app/javascript/alchemy_admin/link_dialog.js

```diff
@@ -0,0 +1,132 @@
+// Represents the link Dialog that appears, if a user clicks the link buttons
+// in TinyMCE or on an Ingredient that has links enabled (e.g. Picture)
+//
+export class LinkDialog extends Alchemy.Dialog {
+  constructor(link) {
+    const url = new URL(Alchemy.routes.link_admin_pages_path, window.location)
+    const parameterMapping = {
+      url: link.url,
+      selected_tab: link.type,
+      link_title: link.title,
+      link_target: link.target
+    }
+
+    // searchParams.set would also add undefined values
+    Object.keys(parameterMapping).forEach((key) => {
+      if (parameterMapping[key]) {
+        url.searchParams.set(key, parameterMapping[key])
+      }
+    })
+
+    super(url.href, {
+      size: "600x320",
+      title: "Link"
```

Comment:
This needs to be translated

```suggestion
      title: Alchemy.translate("Link")
```